packages/common/src/content/_internal/getPropertyMap.ts

```diff
@@ -0,0 +1,22 @@
+import { IPropertyMap } from "../../core/_internal/PropertyMapper";
+import { getBasePropertyMap } from "../../core/_internal/getBasePropertyMap";
+
+/**
+ * Returns an Array of IPropertyMap objects
+ * that define the projection of properties from a IModel to an IHubProject
+ * @returns
+ * @private
+ */
+
+export function getPropertyMap(): IPropertyMap[] {
+  const map = getBasePropertyMap();
+
+  /**
+   * content-specific mappings. Note: we do not need to explicitly map
```

Comment:
In all of our other entity-specific `getPropertyMap` functions, we are mapping the following for `permissions` and `capabilities`: 

```ts
map.push({ objectKey: "permissions", modelKey: "data.permissions" });
map.push({ objectKey: "capabilities", modelKey: "data.settings.capabilities" });
```

so that `permissions` and `capabilities` are at the top-level of the entity object. So I'm wondering: 

1. Should you add that here? 
2. Should we move that to the `getBasePropertyMap` since it appears to be consistent between entities that extend the `IHubItemEntity`?